config/config.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/rs/zerolog"
 
 	"github.com/cloudradar-monitoring/plexus/control"
+	"github.com/cloudradar-monitoring/plexus/pcpairing"
```

Comment:
Why did you rename this to pcpairing? IMO pairing was better, as you could possibly use the paring functionality on a server.